runtime/ms-rest/lib/serviceClient.js

```diff
@@ -52,6 +53,7 @@ function ServiceClient(credentials, options) {
   options.filters.push(RedirectFilter.create());
   if (!options.noRetryPolicy) {
     options.filters.push(new ExponentialRetryPolicyFilter());
+    options.filters.push(new SystemErrorRetryPolicyFilter());
```

Comment:
@huangpf - the retry filter is added by default. So no one needs to add extra code unless they are not satisfied by the default scenario.